src/components/container/ProviderSearch/ProviderSearch.tsx

```diff
@@ -12,10 +12,12 @@ import { FontAwesomeSvgIcon } from 'react-fontawesome-svg-icon';
 export interface ProviderSearchProps {
     previewState?: ProviderSearchPreviewState;
     providerCategories?: string[];
+    onProviderSelected?: (provider: ExternalAccountProvider) => void;
     onProviderConnected?: (provider: ExternalAccountProvider) => void;
-    innerRef?: React.Ref<HTMLDivElement>
-    connectExternalAccountOptions?: ConnectExternalAccountOptions,
-    hideRequestProviderButton?: boolean
+    innerRef?: React.Ref<HTMLDivElement>;
+    connectExternalAccountOptions?: ConnectExternalAccountOptions;
+    hideRequestProviderButton?: boolean;
```

Comment:
@vladtar How are the providers here different from what is returned from getExternalAccountProviders? 
Is this something we anticipate component users to be customizing regularly?

Reply:
I don't think component users will really be customizing this much; it has a fairly narrow use case of just the new welcome screen.

We might add more to the existing API or customize it based on user data in the future, so I don't think it makes sense to have it always use the public one